### PR TITLE
feat(goreleaser): update spec

### PR DESCRIPTION
**Pull request by caarlos0**
PR generated automatically from push-to-fig-autocomplete-action.

### CODE REVIEW

File: src/goreleaser.ts [hunk 1]
```diff
@@ -18,239 +16,315 @@ const configOption: Fig.Option = {
...
-        configOption,
-        rmDistOption,
-        parallelismOption,
-        skipValidateOption,
-        snapshotOption,
-        timeoutOption,
+        {
+          name: ["--config", "-f"],
+          description: "Load configuration from file",
+          args: { name: "config" },
```
Comment (fedeci) [inline src/goreleaser.ts:57]: Filepaths is likely missing in the goreleaser codebase:
```suggestion
          args: { name: "config", template: "filepaths" },
```

File: src/goreleaser.ts [hunk 2]
```diff
@@ -18,239 +16,315 @@ const configOption: Fig.Option = {
...
-          },
+          description: "Builds only the specified build ids",
+          isRepeatable: true,
+          args: { name: "id" },
+        },
+        {
+          name: ["--output", "-o"],
+          description:
+            "Copy the binary to the path after the build. Only taken into account when using --single-target and a single id (either with --id or if configuration only has one build)",
+          args: { name: "output" },
```
Comment (fedeci) [inline src/goreleaser.ts:74]: Same here:
```suggestion
          args: { name: "output", template: "filepaths" },
```

File: src/goreleaser.ts [hunk 3]
```diff
@@ -18,239 +16,315 @@ const configOption: Fig.Option = {
...
         {
-          name: ["-o", "--output"],
-          description: 'Where to save the json schema (default "-")',
-          args: {
-            name: "outputPath",
-            template: "filepaths",
-          },
+          name: ["--output", "-o"],
+          description: "Where to save the json schema",
+          args: { name: "output", default: "-" },
```
Comment (fedeci) [inline src/goreleaser.ts:190]:
```suggestion
          args: { name: "output", default: "-", template: "filepaths" },
```

**Review by fedeci — APPROVED** [review]
Thanks @caarlos0! I am merging this and making some polishing changes! I am leaving some suggestions for the goreleaser codebase hereby in the PR comments (yeah fig autogenerated specs are great to catch possible improvements too)!

**Comment by caarlos0** [comment]
thanks @fedeci , opened  goreleaser/goreleaser#3444 which I think should fix this :)

**Pull request merged**